### PR TITLE
First draft of how to run UPS with Operator

### PR DESCRIPTION
**Pull request by Aindriunannerb**
## Motivation
https://issues.redhat.com/browse/AEROGEAR-10145

## What
Details how to install  UPS operator and use it to install UnifiedPush Server on Openshift.

### CODE REVIEW

File: website/docs/running.md [hunk 1]
```diff
@@ -123,7 +123,113 @@ podman run -p 8080:8080 --rm \
 ```
 
 ## Running with Operator
-- AeroGear 10145
 
+_UnifiedPush_ can be easily installed and maintained on OpenShift by using the _UnifiedPush Operator_. This Operator makes managing the UnifiedPush Server and Database a seamless process.There are a number of prerequisites to getting started with the UnifiedPush Operator.
+
```
Comment (secondsun) [inline website/docs/running.md]: This is only necessary for development.  If you don't want to use the compiler and friends you can just apply the yaml files in the deploy directoy, and the crd files in deploy/crd.

Comment (secondsun) [reply website/docs/running.md]: These could be moved to a section on development.

**Comment by ziccardi** [comment]
Moved here: https://github.com/aerogear/aerogear.org/pull/789/files

**Comment by ziccardi** [comment]
@Aindriunannerb @secondsun Is this still a draft? Can we merge this?

**Pull request merged**